force-app/main/default/objects/Trailer__c/validationRules/Trailer_Number_Validation.validationRule-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Trailer_Number_Validation</fullName>
+    <active>true</active>
+    <errorConditionFormula>OR(Trailer_Number__c &lt; 1,Trailer_Number__c &gt;1000)</errorConditionFormula>
+    <errorMessage>Trailer number should be between 1 and 1000.</errorMessage>
```

Comment:
Do you want to keep in DB so many trailes for each Tv serie? Consider lower number of max number

Reply:
Ok, I changed it to 10. I considered that usually for one season is one trailer so from that was that number. I suppose that normally we get or ask for that type of information.